src/redprl/refiner_types.fun

```diff
@@ -2459,15 +2440,14 @@ struct
 
   structure Universe =
   struct
-    open Universe
-
-    (* XXX needs double-checking *)
-    val kindConstraint =
+    val inherentKind =
       fn K.DISCRETE => K.DISCRETE
        | K.KAN => K.KAN
-       | K.HCOM => K.KAN
-       | K.COE => K.STABLE
-       | K.STABLE => K.STABLE
+       | K.HCOM => K.COE
+       | K.COE => K.COE
+       | K.STABLE => K.COE
```

Comment:
I'll take your word for it :wink:

Reply:
I am just moving the function down. `git` failed us.